controller/api/public/prometheus.go

```diff
@@ -176,7 +181,7 @@ func (s *grpcServer) getPrometheusMetrics(ctx context.Context, requestQueryTempl
 	// kick off asynchronous queries: request count queries + 3 latency queries
 	for pt, requestQueryTemplate := range requestQueryTemplates {
 		var query string
-		if pt == promTCPConnections {
+		if pt == promTCPConnections || pt == promGatewayAlive || pt == promNumMirroredServices {
```

Comment:
I think this was a bit of a hack to begin with and is getting worse here.  What if we added a boolean `timeWindow` parameter to the signature of `getPrometheusMetrics` that indicates if the query expects a time window?

Reply:
I wonder, can we jsut check whether the time window is empty or not? We should always have a default time window if the query needs one, correct ?

Reply:
makes sense to me